modules/runtime/pages/tomcat-bundle.adoc

```diff
@@ -54,7 +54,6 @@ The Tomcat bundle is based on a standard Tomcat installation with the following
 * `server/webapps/bonita`: the Bonita web application.
 * `setup/`: a tool to manage Bonita Platform configuration, stored in database instead of filesystem. Also ships a tool to centralize all the required Tomcat bundle configuration.
 * `tools/request_key_utils: folder containing the script to generate license request keys (Subscription editions only).
```

Comment:
```suggestion
* `setup/`: a tool to manage Bonita Platform configuration, stored in database instead of filesystem. Also ships a tool to centralize all the required Tomcat bundle configuration.
* `tools/request_key_utils`: folder containing the script to generate license request keys (Subscription editions only).
```